project2/README.md

```diff
@@ -53,14 +53,14 @@ pipenv run lint
 `py/main.py` takes an AllRecipes url from standard input and prints a transformed recipe to standard ouput. Eg:
 
 ```sh
-python py/main.py <transform_code> < address
+python py/main.py {transform_code} < address
 ```
 
 ### Transform Codes
 
 - `veg` - to vegetarian
-- `meat*` - from vegetarian
-- `healthy*` - to healthy
-- `unhealthy*` - from healthy
+- `meat` - from vegetarian
+- `healthy` - to healthy
+- `unhealthy` - from healthy
 
 \* not yet supported
```

Comment:
We can remove the not yet supported comment now, right?

Reply:
yessir